install/ansible/cri-containerd.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- hosts: all 
+  become: true
+  # Setting the environment variables that might be needed for the entire installation
+  environment:
+    PATH: "{{ ansible_env.PATH }}:/opt/golang/bin:/usr/local/go/bin:/usr/local/bin"
+    GOPATH: "/opt/golang"
+    GOBIN: "/opt/golang/bin"
+  tasks:
+    - include_vars: vars/vars.yaml # Contains tasks variables for installer
+    - include: tasks/bootstrap_ubuntu.yaml #Contains tasks bootstrap components for ubuntu systems 
+      when: ansible_distribution == "Ubuntu"
+    - include: tasks/bootstrap_centos.yaml #Contains tasks bootstrap components for centos systems
+      when: ansible_distribution == "CentOS"
+    - include: tasks/k8.yaml #Contains tasks kubernetes component installation
+    - include: tasks/build.yaml #Contains tasks for pulling containerd and cri-containerd components
```

Comment:
This is not `build`, right? We just download and extract tar.

Reply:
well it was more of binary builds being pulled :) Couldn't think of a better term.

Reply:
* `install-cri-containerd.yaml` ?
* `install.yaml` ?
* `cri-containerd.yaml` ?

Bad at naming. :P But `build` does not properly reflect what the script is doing.